app/helpers/submissions_helper.rb

```diff
@@ -240,6 +239,18 @@ def sanitize_file_name(file_name)
         SubmissionFile::SUBSTITUTION_CHAR)
   end
 
+  # Helper methods to determine remark request status on a submission
+  def remark_in_progress(submission)
+    submission.remark_result && 
+    submission.remark_result.marking_state == Result::MARKING_STATES[:partial]
+  end
+
+  def remark_complete_but_unreleased(submission)
+    submission.remark_result &&
+    submission.remark_result.marking_state == Result::MARKING_STATES[:complete] &&
```

Comment:
Line is too long. [82/80]<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.